### ISSUE TITLE
data_kidney_KPMP_VISIUM

### ISSUE THREAD

**Issue by FADHLyemen** [SpatialHackathon/SpaceHack2023#101]

**Comment by FADHLyemen** [comment]
@Jieran-S 
Yes, I already implemented it and tested it.

**Comment by Jieran-S** [comment]
Awesome!! I couldn't find the folder for your data at the main branch. Could you point me to the folder for your results? Thank you!

**Comment by FADHLyemen** [comment]
`/home/jovyan/scratch/SpaceHack2/data/KPMP_KIDNEY`

download script is
`/home/jovyan/scratch/SpaceHack2/data/KPMP_KIDNEY/downLoad_KPMP_data.ipynb`

test scrip is
`'scratch/SpaceHack2/test_scripts/test_data_wo_download_KPMP.sh'`

**Comment by Jieran-S** [comment]
Yes! Thank you! I might do some modifications so we can loop them on github as well --- Will discuss this with Naveed and keep you updated! Thanks for implementing them :))
